frontend/src/relay/auth.tsx

```diff
@@ -49,7 +50,7 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
 }) => {
     const target = serviceUrl(service);
     for (const [key, value] of Object.entries(params)) {
-        target.searchParams.append(key, value);
+        target.searchParams.append(key, value as string);
```

Comment:
Hu, why is that necessary now? Before, the type of `value` was probably `URL`? And now it's `string | URL`? I don't understand why a cast is necessary now...

Reply:
The heck. Yeah I just checked locally. Why would it infer the type to be `any` 0.o

Reply:
Ehm but ok. It's probably a good idea to change `URL` to `string`, as the `as string` cast is just a typescript thing and results in no actual runtime code, i.e. it does not change any values. Apparently the URLs `toString` method is used or sth. But yeah, we can probably do that.